projects/fsastorefrontlib/src/cms-components/comparison-table/comaprison-table-panel-item/comparison-table-panel-item.component.html

```diff
@@ -1,6 +1,9 @@
 <ng-container *ngIf="product$ | async as product; else loading">
   <ng-container *ngIf="!isLoading; else loading">
-    <div class="table-header">
+    <div
+      class="table-header"
+      [ngClass]="recommendedProduct == product.code ? 'highlight' : ''"
```

Comment:
Or maybe to try with syntax:

[class.highlight]="recommendedProduct == product.code"